piapage/repairs.py

```
@@ -8,21 +8,22 @@ def repair_tables(missions, hosts, instruments, targets, systems):
+    if not systems:
+        systems = ['']
+
+    targets = ['Sun' if t == 'Sol' else t for t in targets]
+    if 'Sun' in targets and 'Sun' not in systems:
+        if systems == ['']:
+            systems = ['Sun']
+        else:
+            systems.append('Sun')
 
     if 'Rosetta' in missions:
         if targets[0] == 'Comet':
```

Ruff (0.15.9)

[warning] Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)